<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ValueDialog</class>
 <widget class="QDialog" name="ValueDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="_label">
       <property name="text">
        <string>格式化: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="_radioButtonText">
       <property name="text">
        <string>text</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="_radioButtonJson">
       <property name="text">
        <string>json</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="_radioButtonXml">
       <property name="text">
        <string>xml</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="_radioButtonHtml">
       <property name="text">
        <string>html</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="_horizontalSpacerButton">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>78</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTextEdit" name="_valueTextEdit"/>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="_horizontalSpacerFormat">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>118</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="_pushButtonY">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="_pushButtonN">
       <property name="text">
        <string>关闭</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
